loaders/mongooseLoader.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+async function connectToMongoDB() {
+  try {
+    await mongoose.connect(process.env.MONGO_DB_URL, () => {
+      console.log("Succesfully connect to mongoDB");
```

Comment:
개인적으로 느끼기엔 계속 `mongoDB`로 작성하니깐 upper case일때도 붙여주면 좋을 것 같네요!
```suggestion
    await mongoose.connect(process.env.MONGODB_URL, () => {
      console.log("Successfully connect to mongoDB");
```